utils_nlp/models/transformers/named_entity_recognition.py

```diff
@@ -1,2 +1,430 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
\ No newline at end of file
+# Licensed under the MIT License.
+
+import torch
+import logging
+import numpy as np
+
+from torch.utils.data import TensorDataset
+from cached_property import cached_property
+from collections import Iterable
+
+from transformers.modeling_bert import (
+    BERT_PRETRAINED_MODEL_ARCHIVE_MAP,
+    BertForTokenClassification
+)
+
+from utils_nlp.models.transformers.common import (
+    MAX_SEQ_LEN,
+    TOKENIZER_CLASS,
+    Transformer,
+    get_device,
+)
+
+TC_MODEL_CLASS = {k: BertForTokenClassification for k in BERT_PRETRAINED_MODEL_ARCHIVE_MAP}
```

Comment:
no need to prefix with 'TC'

Reply:
There is a same MODEL_CLASS in Sequence Classification. Since it's a global definition, I just don't want to have a conflict name with the one in Sequence Classification. Another reason is it could be used to control what Transformer models is supported (right now only BERT).